contracts/facets/EmbalmerFacet.sol

```diff
@@ -79,68 +75,93 @@ contract EmbalmerFacet {
             );
         }
 
+        // Validate exactly 2 arweave TX IDs have been provided
+        if (arweaveTxIds.length != 2 || bytes(arweaveTxIds[0]).length == 0 || bytes(arweaveTxIds[1]).length == 0) {
+            revert LibErrors.ArweaveTxIdsInvalid();
+        }
+
         // Confirm that archaeologists are provided
         if (selectedArchaeologists.length == 0) {
             revert LibErrors.NoArchaeologistsProvided();
         }
 
+        // Confirm that minShards is greater than 0
+        if (sarcophagus.minShards == 0) {
+            revert LibErrors.MinShardsZero();
+        }
+
         // Confirm that minShards is less than the number of archaeologists
         if (sarcophagus.minShards > selectedArchaeologists.length) {
             revert LibErrors.MinShardsGreaterThanArchaeologists(
                 sarcophagus.minShards
             );
         }
 
-        // Confirm that minShards is greater than 0
-        if (sarcophagus.minShards == 0) {
-            revert LibErrors.MinShardsZero();
-        }
-
         // Initialize a list of archaeologist addresses to be passed in to the
         // sarcophagus object
-        address[] memory archaeologistsToBond = new address[](
+        address[] memory cursedArchaeologists = new address[](
```

Comment:
I renamed this only b/c the `archaeologistsToBond` variable is used **after** the archs have already been bonded (as well as before).